### PR TITLE
contrib: fix mk-tls-keypair.sh

### PR DESCRIPTION
**Pull request by mwasilew**
This patch adds missing x509v3 values to the config files.

### CODE REVIEW

File: contrib/mk-tls-keypair.sh [hunk 1]
```diff
@@ -50,13 +51,14 @@ openssl req -new -config ${conf} -key ${keyfile} -out ${csrfile}
 dns=$(openssl req -text -noout -verify -in $csrfile | grep DNS:)
 echo "signing with dns name: $dns" 1>&2
 cat >${tmpdir}/server.ext <<EOF
-keyUsage=critical, digitalSignature, keyEncipherment, keyAgreement
+keyUsage=critical, digitalSignature, keyEncipherment, keyAgreement, keyCertSign
 extendedKeyUsage=critical, serverAuth
 subjectAltName=$dns
+basicConstraints = critical, CA:true, pathlen:0
```
Comment (doanac) [inline contrib/mk-tls-keypair.sh:57]: this script is for creating a TLS cert - not a CA.

Comment (mwasilew) [reply contrib/mk-tls-keypair.sh:57]: from what I understand it's a CA (just intermediate). I'm updating it to the factory with:
```
fioctl keys ca update
```
And the factory is demanding that the `keyCertSign` is in the cert.

Comment (doanac) [reply contrib/mk-tls-keypair.sh:57]: there are two keys here and you are mixing them up.

1) An intermediate CA to sign requests with
2) A TLS cert for web traffic

Our backend needs to know about 1 so our device-gateway can trust devices signed with certs it produces. This script is creating 2.

**Comment by mwasilew** [comment]
This PR is wrong. Closing

**Pull request closed without merging**